test/integration/backward_compatible/topic/ReliableTopicOnClusterRestart.js

```diff
@@ -31,12 +31,12 @@ describe('ReliableTopicOnClusterRestartTest', function () {
 
     before(async function () {
         cluster = await RC.createCluster(null, null);
-        member = await RC.startMember(cluster.id);
     });
 
     beforeEach(async function () {
         client1 = undefined;
         client2 = undefined;
+        member = await RC.startMember(cluster.id);
```

Comment:
I think to make sure that the tests do not affect each other we should move `cluster = await RC.createCluster(null, null);` to beforeEach (and its shutdown to afterEach) as well

Reply:
Makes sense, done